x/bank/internal/invariants/msgs.go

```diff
@@ -1,4 +1,4 @@
-package simulation
```

Comment:
this is not consistent with what the other modules have. Can we come to consensus which is the standard for this package ?

Reply:
`simulation` is now gone

Reply:
ok, updated #4438

Reply:
I don't think this was a wise choice. Simulation messages have nothing to do with invariants.